### ISSUE TITLE
Can the Godot package use Install-BinFile so the shim exe is simply godot.exe?

### ISSUE THREAD

**Issue by ferventcoder** [joachimschmidt557/chocolatey-packages-manual-automatic#3]
https://chocolatey.org/packages/godot#files

The package for Godot unpacks a binary that get shimmed automatically. However given the name, something like Godot_v3.1-stable_win64.exe, it's not very guessable and easy to call. It would be much better to use [Install-BinFile](https://chocolatey.org/docs/helpers-install-bin-file) to get a shim exe named godot.exe?

The documentation for Install-BinFile is at https://chocolatey.org/docs/helpers-install-bin-file, and it's pretty straightforward to implement.

It should be something about like the following after you call Get-ChocolateyUnzip:

~~~powershell
$fileToShim  = Get-Item "$toolsDir\Godot*.exe"
Install-BinFile -Name 'godot' -Path "$fileToShim"`
~~~

Also: If you want to not have Chocolatey shim the longer named file, you can create a shimgen ignore file at runtime after you unpack the longer named one and Chocolatey won't shim it. The ignore file must sit right next to the file you want shimgen to ignore and it must match in case and everything. The name of the file is exactly the same as the exe with `.ignore` appended at the end. So if your exe is named `bob.exe`, the shimgen ignore file will be named `bob.exe.ignore`.

**Comment by ferventcoder** [comment]
If you want to not have Chocolatey shim the longer named file, you can create a shimgen ignore file at runtime after you unpack the longer named one and Chocolatey won't shim it. The ignore file must sit right next to the file you want shimgen to ignore and it must match in case and everything. The name of the file is exactly the same as the exe with `.ignore` appended at the end. So if your exe is named `bob.exe`, the shimgen ignore file will be named `bob.exe.ignore`.

**Comment by joachimschmidt557** [comment]
Ok, I'll work on this. Thanks for notifying me.

**Comment by J534** [comment]
Thank you for looking into this!

**Comment by joachimschmidt557** [comment]
@ferventcoder Should I also create a `chocolateyUninstall.ps1` file which calls `Uninstall-BinFile` to remove the shim again?

**Comment by ferventcoder** [comment]
If you are adding one for the install, then you should remove it as part of the uninstall, yes.

**Comment by joachimschmidt557** [comment]
This was implemented as of 834f744db071d3169ef6116405ca5aaa07f0c303
